python/ct/client/log_client.py

```diff
@@ -197,8 +199,10 @@ def _parse_consistency_proof(response, servername):
 class RequestHandler(object):
     """HTTPS requests."""
     def __init__(self, connection_timeout=60, ca_bundle=True):
-      self._timeout = connection_timeout
-      self._ca_bundle = ca_bundle
+        if isinstance(ca_bundle, str):
```

Comment:
It wasn't clear to me why it was OK for a parameter to be either true/false or a string.
I'll make the change not least because the result will be a lot nicer.